mlflow/models/flavor_backend_registry.py

```diff
@@ -13,6 +13,9 @@ def get_flavor_backend(model, build_docker=True, **kwargs):
     from mlflow.pyfunc.backend import PyFuncBackend
     from mlflow.rfunc.backend import RFuncBackend
 
+    if not model:
+        return pyfunc.FLAVOR_NAME, PyFuncBackend({}, **kwargs)
```

Comment:
You don't set flavor config here, will it cause issues ? `PyFuncBackend(flaver_configs, **kwargs)` ?

Reply:
It probably doesn't. `flaver_configs` is not required when building an image.

Reply:
fine.

Reply:
@harupy Looks like we default to `PyfuncBackend` here though. Will users be able to serve models with RFunc / Java MLeap model server after building this generic image?

Reply:
> Will users be able to serve models with RFunc / Java MLeap model server after building this generic image?

They won't. Only pyfunc model is supported now. Do we need to support serving them?